controller/backup_volume_controller.go

```diff
@@ -284,11 +284,27 @@ func (bvc *BackupVolumeController) reconcile(backupVolumeName string) (err error
 		return err
 	}
 
+	// Get `Time to live` after failed backup was marked as `Error` or `Unknown`,
+	// default: 180s and it is shorter than `Backupstore Poll Interval` by default, so failed backups should be deleted once the backup volume is updated.
+	failedBackupTTL, err := bvc.ds.GetSettingAsInt(types.SettingNameFailedBackupTTL)
+	if err != nil {
+		log.WithError(err).Errorf("failed to get %v setting from cluster", types.SettingNameFailedBackupTTL)
```

Comment:
"..., will skip the auto-deletion for the failed backups"

Reply:
Added. Thanks.